unfoldingWord/translationCore
# Build off branches

#### This pull request addresses:

Finishes #3203. This refactors the build process to generate builds when certain branches receive updates.

* `develop`
* `release-*`
* `sandbox-*`
* and all tags

#### How to test this pull request:

You can see an overview of builds at http://tc.unfoldingword.surge.sh. If you push up a branch following the above naming conventions the appropriate web page will be updated to reflect the successful build and distributables will be available for download.

I have personally tested the sandbox and release branches already. However, the only way to test the develop branch builds is to merge in this PR.
Likewise in order to test builds on tags we'll need to tag a new release.

For a complete description of the build process see the [New DSM dev team](https://docs.google.com/document/d/1-dJvDrxjIYwiUKUnH-E8qZel-DCDhsgM8iR-n9Cm57w/edit#) document.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3298)
